besu/src/main/java/org/hyperledger/besu/controller/CliqueBesuControllerBuilder.java

```diff
@@ -132,7 +132,6 @@ protected ProtocolSchedule createProtocolSchedule() {
         genesisConfigOptions,
         forksSchedule,
         nodeKey,
-        privacyParameters,
```

Comment:
why is it safe to use this constructor which uses default privacy parameters rather than the superclass' privacy parameters which may (or may not) have been set on the CLI